installing_gitops/installing-openshift-gitops.adoc

```diff
@@ -38,3 +38,8 @@ include::modules/installing-gitops-operator-using-cli.adoc[leveloffset=+1]
 
 // Logging in to the Argo CD instance by using the Argo CD admin account
 include::modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_installing-openshift-gitop"]
```

Comment:
```suggestion
[id="additional-resources_installing-openshift-gitops"]
```

Reply:
I apologize for suggesting the ID with the spelling error, this is the right ID. Thank you!

Reply:
Fixed.